### ISSUE TITLE
Infer logprob of absolute operations 

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc#6402]
### Description

https://github.com/pymc-devs/pymc/pull/6400 extended the logprob inference to 2-to-1 transforms. It's now straightforward to add an Absolute transform to infer the logprob of cases like

```python
import pymc as pm

x = pm.math.abs(pm.Normal.dist())
y = pm.HalfNormal.dist()
assert pm.logp(x, 2.5).eval() == pm.logp(y, 2.5).eval()
```

**Comment by LukeLB** [comment]
I'm taking a look at this, but just wanted to check my understanding...

So similar to what was done in #6400, there needs to be an `AbsoluteTransform()` class created and another if statement added to `find_measurable_transforms()` to check for whether the Elemwise operator is an absolute transform and then set the transform variable to the created `AbsoluteTransform()`. 

Am I missing anything?

**Comment by ricardoV94** [comment]
@LukeLB that's it!

**Comment by LukeLB** [comment]
I'll get to it then!

**Comment by ricardoV94** [comment]
Closed via #6414
